apps/roam/src/utils/hyde.ts

```diff
@@ -0,0 +1,375 @@
+import { Result } from "./types";
```

Comment:
@maparent We don't want any front end changes to be merged into main.
These changes are also diverged from what will be eventually merged from `eng-233-suggestive-mode-internal`, so I feel we'll have the same nightmarish rebase/merge problem.